.github/workflows/prev_deploy.yml

```diff
@@ -0,0 +1,34 @@
+name: Preview Deployment
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+  pull_request:
+    types: [opened]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      deployments: write
+      pull-requests: write
+    steps:
+      - name: Await CF Pages
+        uses: WalshyDev/cf-pages-await@v1
+        id: pages-action
+        with:
+          apiToken: ${{ secrets.CF_API_TOKEN }}
+          accountId: '9793bc27d30f5d738baf5e5a491b7416'
```

Comment:
there is the account id hardcoded and seems to be corresponding to you personal account on cloudflare. And we added the project under your sixsq email. should you update that value ?

Reply:
uff yes, thanks, that was the problem 😄